libbeat/docs/command-reference.asciidoc

```diff
@@ -17,11 +17,11 @@
 :deploy-command-short-desc: Deploys the specified function to your serverless environment
 
 ifndef::no_dashboards[]
-:export-command-short-desc: Exports the configuration, index template, or a dashboard to stdout
+:export-command-short-desc: Exports the configuration, index template, ilm-policy, or a dashboard to stdout
```

Comment:
Global comment: This section uses ilm-policy, ilm policy, and ILM policy inconsistently. Except when you are referring to the actual option name, you should say ILM policy (because abbreviations are always capitalized).

Reply:
good catch - I changed it.